FordLabs/PeopleMover
# Space name fit on the dashboard modal

## Issue
Connects #280 

## What was done
- [x] Added 40 character limit to create space form
- [x] Space name wraps text to a second line in the modal instead of spilling over
- [x] Spaces can now have lowercase letters, special characters, and spaces

## How to test
1. Create a new space and attempt to add more than 40 characters. If space name is long, see that the space name in the space dashboard goes across multiple lines instead of spilling over. Also use spaces, special characters, and lowercase letters for space name and see that reflected on the tiles.
